docs/UsingCppInterOp.rst

```diff
@@ -65,35 +65,255 @@ robust (simple function calls, no inheritance, etc.). The goal is to make it as
 close to the compiler API as possible, and each routine should do just one thing.
 that it was designed for.
 
-Further Enhancing the Dynamic/Automatic bindings in CPPYY
-=========================================================
-The main use case for CppInterOp is the CPPYY service. CPPYY is an
-automatic run-time bindings generator for Python and C++, and supports a wide
-range of C++ features (e.g., template instantiation). It operates on demand and
-generates only what is necessary. It requires a compiler (Cling or Clang-REPL).
-that can be available during programme runtime.
+How cppyy leverages CppInterOp
+===============================
+
+CppInterOp supports tools that utilize interactive C++ by using the compiler as
```

Comment:
given the title, the focus here should be on cppyy. So, start with that and connect to CppInterOp instead of the other way around

Reply:
updated text:
cppyy is a run-time Python-C++ bindings generator for calling C++ from Python
and Python from C++. Interestingly, it uses C++ interactively by using the
compiler as a service. This is made possible by the CppInterOp library.
Following are some of the ways cppyy leverages CppInterOp for better
performance and usability.